stack/base-with-services/before-notebook.d/30_start-rabbitmq.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -emx
+
+# Fix issue where the erlang cookie permissions are corrupted.
+chmod 400 "/home/${NB_USER}/.erlang.cookie" || echo "erlang cookie not created yet."
+
+mamba run -n aiida-core-services rabbitmq-server -detached
```

Comment:
I don't understand, for arm64, the rabbitmq-server is not installed from conda. I guess this will fail for the arm64.
This script will run both for the amd64 and arm64 I assume. Originally, the script is different copied based on the arch.

Reply:
Very good catch, not sure how I missed it, thanks. I did trigger an arm64 test originally but have removed those commits when rebasing. Will test again.

Reply:
I think this might actually work, since in arm64 rabbitmq is available globally, so it doesn't matter whether it is run through conda environment or not.

Reply:
The tests passed so I think this approach works. @unkpcz if you have a Mac you can test manually if you want. 

I added a note with an explanation. Hopefully soon we'll have rabbitmq conda package for arm64 as well

Reply:
Okay, thanks for checking. All good for me then.